vertical-pod-autoscaler/pkg/utils/test/test_utils.go

```diff
@@ -89,6 +90,16 @@ func Resources(cpu, mem string) apiv1.ResourceList {
 	return result
 }
 
+// AddResource add a resource to the given resource list
+func AddResource(rl apiv1.ResourceList, resourceName apiv1.ResourceName, value string) apiv1.ResourceList {
```

Comment:
There's only one place we're using this function. Please move it to vertical-pod-autoscaler/pkg/utils/test/test_recommendation.go and unexport it (rename to `addResource`)

Reply:
done